### DISCUSSION TITLE
Audio Gridder doesn't work in Wavelab Pro 11

### DISCUSSION THREAD

**Discussion by falonite** [apohl79/audiogridder#1156] [Help]
The Audio Gridder plugin launches in the DAW, however, even though the plugin reads the server, it won't connect to it, the audio window that usually loads upon connection also doesn't show.

**Comment by jterrain** [comment]
Confirmed in Steinberg WaveLab Pro 12.0.51 on Windows. 

Until the WaveLab playback transport is started, AudioGridder plugin autodetects and shows correct server name and IP, but refuses to connect to the server; but in other DAWs in the same system will connect easily. Once WaveLab playback is started, the server connection is successfully made.

No special network/firewall rules exist for WaveLab in this case.
